lib/ston_api_client.ex

```diff
@@ -1,18 +1,53 @@
 defmodule StonApiClient do
   @moduledoc """
-  Documentation for `StonApiClient`.
+  A client for working with the API of a decentralized automated market maker Ston.fi
   """
 
+  alias StonApiClient.Asset
+
+  @base_url "https://api.ston.fi/v1"
+
   @doc """
-  Hello world.
+  Getting information about all assets
+  """
+  def assets do
+    with {:ok, data} <- make_request("#{@base_url}/assets") |> parse_body,
```

Comment:
По негласному соглашению, скобки у функций указываются всегда, а пайпы состоят из 2+ вызовов и начинаются с аргумента.
```suggestion
    with {:ok, data} <- "#{@base_url}/assets" |> make_request() |> parse_body(),
```